test/loginSuite.robot

```diff
@@ -1,6 +1,16 @@
 *** Settings ***
 Resource          ../Service/LoginService.robot
+Library           Selenium2Library
+
+*** Variables ***
 
 *** Test Cases ***
 1. Login Test
-    Login And Wait Top Logo Image UI
+    OpenBrowser    ${CMP_ADRESS}    gc
```

Comment:
move low-level selenium keywords into relevant pages and services. Use structure I created yesterday